bin/run-all-in-docker.sh

```diff
@@ -1,11 +0,0 @@
-#! /bin/bash -e
-
-SOLUTION_DIR="${1:-dockerTest}"
-
-# build docker image
-docker build --rm --no-cache -t swift-test-runner .
-
-docker run \
```

Comment:
There were several options missing here, including a key one: `--network none`. Other test runners have this file named `run-all-tests-in-docker`, so I've renamed the file and added the right flags that match how the test runner is actually run on the website.